onnxruntime/core/providers/js/js_export.cc

```diff
@@ -24,3 +24,8 @@ const void* JsepOutput(void* context, int index, void* data) {
   LOGF_DEFAULT(VERBOSE, "JsepOutput -- data=%zu", (size_t)(r));
   return r;
 }
+
+const void* JsepGetNodeName(void* kernel) {
+  auto& name = reinterpret_cast<onnxruntime::OpKernel*>(kernel)->Node().Name();
```

Comment:
Is const qualifier required here?

Reply:
you are right, the const qualifier should be here.

The reason why it still works is because the 2 exported functions are called from JS so any pointer is just a number. But it's better to use const here